plugins/woocommerce-admin/package.json

```diff
@@ -42,14 +42,14 @@
 		"start": "pnpm run install-if-deps-outdated && cross-env WC_ADMIN_PHASE=development pnpm run build:packages && cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"cross-env WC_ADMIN_PHASE=development pnpm run:packages -- start --parallel\"",
 		"start:package": "pnpm run:packages -- start --parallel",
 		"pretest": "pnpm run -s install-if-no-packages",
-		"test": "pnpm nx build @woocommerce/internal-js-tests && pnpm run test:client",
+		"test": "pnpm run test:client",
 		"test-staged": "pnpm run test:client -- --bail --findRelatedTests",
 		"test:client": "jest --config client/jest.config.js",
 		"test:debug": "node --inspect-brk ./node_modules/.bin/jest --config client/jest.config.js --watch --runInBand --no-cache",
 		"test:help": "wp-scripts test-unit-js --help",
 		"test:packages": "pnpm run --filter ../../packages/js/ --filter !api-core-tests test",
 		"test:update-snapshots": "pnpm run test:client -- --updateSnapshot && pnpm run --filter @woocommerce/components test:update-snapshots",
-		"test:watch": "pnpm run test:client -- --watch",
+		"test:watch": "tsc --build || concurrently \"pnpm run test:client -- --watch\" \"pnpm run:packages -- test --parallel -- --watch\"",
```

Comment:
@ObliviousHarmony We've removed `tsc --build || concurrently \"` and `\" \"pnpm run:packages -- test --parallel -- --watch\"` in [this PR](https://github.com/woocommerce/woocommerce/pull/33202/files#diff-d744863f2c4a91f591ce12dfc5a77021799504d9fc83551e6ad3f47d05c3e1b7R52). 🙂

Reply:
I think this is due to rebase?